book/templates/chapter-template/chapter-content/chapter-personal-stories.md

```diff
@@ -0,0 +1,32 @@
+(sectioninitials-filename)=
+# Personal Stories
+
+> A personal story is a case study about an individual who practices the main ideas from your chapter in their work.
+> Personal stories should give readers a first-hand perspective and insight about how they may go about introducing your chapter's concepts in their own work.
+
+> A personal story should be short, engaging, and have a clear call-to-action. 
+> How you format it is up to you, however you may consider one of the following options:
+> - An interview with the subject - for reference, see the [personal story](https://the-turing-way.netlify.app/reproducible-research/vcs/vcs-personal-stories.html) in the Verion Control chapter
+> - A blog-style article about the subject written from their perpective - for reference, see the [personal story](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html) in the Research Data Management Chapter.
+> - A blog-style article written about the subject written from your perspective - in this case, it may be beneficial to include direct quotes from the subject in your article where possible. 
+
+## `Subject Name`: `Story Caption`
+
+> The title of the personal story should start with the name of the subject followed by a short caption that captures the overall message of the story.
+> For example, the [personal story](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html) in the Research Data Management chapter is aptly titled: `Susanna-Assunta Sansone: From FAIR Co-Author to FAIR Doer`
+
+> Regardless of how you choose to format the personal story, ensure that it contains the elements itemised in checklist below. Feel free to use images, external resouces, or code blocks to tell a compelling story.
+> Above all, centre the personal story around how readers are expected to benefit from the subject's experiences
+
+> ### Checklist
```

Comment:
The markdown formatting won't work. So maybe we can drop that at the moment.

Reply:
Thanks @malvikasharan. I have updated the PR to remove the checklist formatting